python/lsst/ts/m2com/controller.py

```diff
@@ -165,10 +165,17 @@ async def _connect(self, timeout):
             if self.are_clients_connected():
                 await asyncio.sleep(1)
             else:
-                await asyncio.gather(
-                    self.client_command.connect(timeout=timeout),
-                    self.client_telemetry.connect(timeout=timeout),
-                )
+
+                try:
+                    await asyncio.gather(
+                        self.client_command.connect(timeout=timeout),
+                        self.client_telemetry.connect(timeout=timeout),
+                    )
+
+                except asyncio.TimeoutError:
+                    self.log.debug("Timeout error when connects to servers.")
```

Comment:
Better: 
```
except asyncio.TimeoutError as err:
    self.log.debug(f"Timeouted when connecting to servers - {self.client_command.host}:{self.client_command.port} and/or {self.client_telemetry.host}:{self.client_telemetry.port}: {str(err)}")

Reply:
Fixed in the commit: [a96128e](https://github.com/lsst-ts/ts_m2com/pull/8/commits/a96128e332e3be4733dff30c0dedd2de2813cad8). Thanks!